iati/guidance_and_support/models.py

```diff
@@ -1,7 +1,70 @@
 from django.db import models
 
+from wagtail.core.fields import StreamField
 from wagtail.core.models import Page
+from wagtail.images.edit_handlers import ImageChooserPanel
 
+from home.models import AbstractContentPage, AbstractIndexPage, IATIStreamBlock
 
-class GuidanceAndSupportPage(Page):
-    pass
+
+class GuidanceAndSupportPage(AbstractContentPage):
+    """A base for the Guidance and Support page."""
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['guidance_and_support.GuidanceGroupPage', 'guidance_and_support.KnowledgebaseIndexPage']
+
+    @property
+    def guidance_groups(self):
+        """Get all GuidanceGroupPage objects that have been published."""
+        guidance_groups = GuidanceGroupPage.objects.child_of(self).live()
+        return guidance_groups
+
+
+class GuidanceGroupPage(AbstractContentPage):
+    """A base for Guidance Group pages."""
+    subpage_types = ['guidance_and_support.GuidanceGroupPage', 'guidance_and_support.GuidancePage']
+
+    feed_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+',
+        help_text='This is the image that will be displayed for the guidance index page on the main guidance and support page. Ignore if this page is being used as a sub-index page.'
+    )
+
+    feed_content = StreamField(IATIStreamBlock(required=False), null=True, blank=True, help_text='A small amount of content to appear on the main page (e.g. bullet points). Ignore if this page is being used as a sub-index page.')
```

Comment:
I'm wondering if there's a better title than `feed` - though I've not found it yet,,, The help text on the field in the CMS is useful, but I have to admit I didn't immediately read it.

Reply:
How about `feed_content` -> `base_page_content`, and `feed_button_text` -> `base_page_link_button_text`

Reply:
or `main_page_*` or `section_page_*`...?

Reply:
I think `base_page` is still not clear enough for CMS users =/

Reply:
@allthatilk suggested some user testing - maybe speak to BAs to ask how they perceive these fields